clients/fossid-webapp/src/main/kotlin/FossIdRestService.kt

```diff
@@ -153,6 +157,19 @@ interface FossIdRestService {
                         val value = if (array.isEmpty()) null else array.first()
                         PolymorphicInt(value)
                     }
+                    JsonToken.START_OBJECT -> {
+                        val mapType = JSON_MAPPER.typeFactory.constructMapType(
+                            LinkedHashMap::class.java,
+                            String::class.java,
+                            String::class.java
+                        )
+                        val map = JSON_MAPPER.readValue<Map<Any, Any>>(p, mapType)
+                        if (map.size != 1) {
+                            error("A map representing a polymorphic integer should have one value!")
+                        }
+                        PolymorphicInt(map.values.first().toString().toInt())
+                    }
+
```

Comment:
Nit: Either this empty line should go away (like it was before), or empty lines between all cases should be added.